dask/dataframe/utils.py

```diff
@@ -585,7 +585,9 @@ def _nonempty_series(s, idx=None):
             data = pd.SparseArray([entry, entry], dtype=dtype)
     elif is_interval_dtype(dtype):
         entry = _scalar_from_dtype(dtype.subtype)
-        data = pd.array([entry, entry], dtype=dtype)
+        data = pd.array(
+            [pd.Interval(entry, entry), pd.Interval(entry, entry)], dtype=dtype
```

Comment:
`pd.array([entry, entry], dtype=pd.IntervalDtype())` was failing to create an IntervalArray like
```
pandas/_libs/interval.pyx in pandas._libs.interval.intervals_to_interval_bounds()

TypeError: type <class 'numpy.int64'> with value 1 is not an interval
```

so I switched to this, but is there any pandas-backwards-compatibility reason why this won't work?

Reply:
I think that's fine, and we should have a CI job with our minimum version.